### PR TITLE
[MTM-53170] update gs related docs

### PR DESCRIPTION
**Pull request by sag-lukjan**

### CODE REVIEW

File: content/enterprise-tenant/managing-tenants-bundle/custom-properties.md [hunk 1]
```diff
@@ -36,4 +36,4 @@ Using the checkbox **Enable Gainsight product experience tracking** a parent ten
 
 On tenant level, the product experience tracking by Gainsight can be disabling by disabling the cookie banner on the **Branding** page, see [Branding](/enterprise-tenant/customization/#branding).
 
-Even if tracking is enabled for a tenant, users must actively accept the tracking of functional cookies, before any functional data on the usage of the platform is tracked, see [Accessing and logging into the platform](/get-familiar-with-the-ui/platform-access).
+Even if tracking is enabled for a tenant, users must actively accept (data will contain [Personally identifiable information](https://support.gainsight.com/PX/Security/Policies/Usage_Tracking_and_Personally_identifiable_information_(PII))) or reject (data will be [anonymized](https://support.gainsight.com/PX/Security/Policies/Usage_Tracking_and_Personally_identifiable_information_(PII)#Anonymize_Identity_Data)) the tracking of functional cookies, before any functional data on the usage of the platform is tracked, see [Accessing and logging into the platform](/get-familiar-with-the-ui/platform-access).
```
Comment (DawidSAG) [inline content/enterprise-tenant/managing-tenants-bundle/custom-properties.md:39]: I would go into this direction:

```
When tracking is activated for a tenant, users are automatically tracked. However, the nature of this tracking depends on user consent. By accepting tracking, users permit the use of Personally Identifiable Information (PII) for tracking purposes. If users decline, their data is anonymized, ensuring privacy, yet tracking continues to capture usage data without personal identifiers. For more details, refer to...
```

File: content/enterprise-tenant/managing-tenants-bundle/custom-properties.md [hunk 2]
```diff
@@ -36,4 +36,4 @@ Using the checkbox **Enable Gainsight product experience tracking** a parent ten
 
 On tenant level, the product experience tracking by Gainsight can be disabling by disabling the cookie banner on the **Branding** page, see [Branding](/enterprise-tenant/customization/#branding).
```
Comment (DawidSAG) [inline content/enterprise-tenant/managing-tenants-bundle/custom-properties.md:37]: I'm not sure about this sentence? 
Maybe: At the tenant level, the product experience tracking by Gainsight can be disabled by turning off the cookie banner on the Branding page. For more information, see...

File: content/enterprise-tenant/managing-tenants-bundle/custom-properties.md (hunk 1 above)
Comment (sag-lukjan) [reply content/enterprise-tenant/managing-tenants-bundle/custom-properties.md]: Done, but what about adding docs for PII and anonymization? On the one hand they can be useful, but on the other hand it is more technical documentation in my opinion and I am not sure it fits here.

https://support.gainsight.com/PX/Security/Policies/Usage_Tracking_and_Personally_identifiable_information_(PII)#Overview
https://support.gainsight.com/PX/Security/Policies/Usage_Tracking_and_Personally_identifiable_information_(PII)#Anonymize_Identity_Data

Comment (DawidSAG) [reply content/enterprise-tenant/managing-tenants-bundle/custom-properties.md]: I wouldn't add it, but @BeateRixen @MWindel could help us decide if it makes sense to add it.

**Review by DawidSAG — APPROVED** [review]

**Pull request merged**